samples/apps/copilot-chat-app/webapi/Controllers/SpeechTokenController.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.AspNetCore.Mvc;
+using SemanticKernel.Service.Config;
+
+namespace SemanticKernel.Service.Controllers;
+
+/// <summary>
+/// Token Response is a simple wrapper around the token and region
+/// </summary>
+public class SpeechTokenResponse
+{
+    public string? token { get; set; }
+    public string? region { get; set; }
+}
+
+[ApiController]
+public class SpeechTokenController : ControllerBase
+{
+    private readonly IConfiguration _configuration;
+    private readonly ILogger<SpeechTokenController> _logger;
+
+    public SpeechTokenController(IConfiguration configuration, ILogger<SpeechTokenController> logger)
+    {
+        this._configuration = configuration;
+        this._logger = logger;
+    }
+
+    /// <summary>
+    /// Use the Azure Speech Config key to return an authorization token and region as a Token Response.
+    /// </summary>
+    [Route("speechToken")]
```

Comment:
What the intended route here? I believe this wil lend up as http://localhost:3000/SpeechToken/speechToken yeah? I recommend renaming the controller to "TokenController" and having this route be "speech", so that would give us `http://localhost:3000/token/speech`.

Reply:
No, the eventual path will be http://localhost:40443/speechToken, regardless of what the class is actually called the commandURL will be the baseURL_forApp + route

Reply:
No, the URL ends up being http://localhost:40443/speechToken, the class name isnt amended to the path, just the route itself.